--- x/etc/hosts	2025-01-12 09:10:20.394349272 +0530
+++ y/etc/hosts	2024-04-02 07:09:25.000000000 +0530
@@ -1,5 +1,9 @@
-127.0.0.1	localhost
-::1		localhost ip6-localhost ip6-loopback
-ff02::1		ip6-allnodes
-ff02::2		ip6-allrouters
+127.0.0.1 localhost

+# The following lines are desirable for IPv6 capable hosts
+::1 ip6-localhost ip6-loopback
+fe00::0 ip6-localnet
+ff00::0 ip6-mcastprefix
+ff02::1 ip6-allnodes
+ff02::2 ip6-allrouters
+ff02::3 ip6-allhosts

--- x/etc/apt/sources.list	2025-01-12 09:31:34.678961593 +0530
+++ y/etc/apt/sources.list	2025-01-03 06:53:06.000000000 +0530
@@ -1 +1,4 @@
-deb http://ports.ubuntu.com/ubuntu-ports oracular main restricted universe multiverse
+# Ubuntu sources have moved to the /etc/apt/sources.list.d/ubuntu.sources
+# file, which uses the deb822 format. Use deb822-formatted .sources files
+# to manage package sources in the /etc/apt/sources.list.d/ directory.
+# See the sources.list(5) manual page for details.

--- x/etc/apt/sources.list.d/ubuntu.sources	1970-01-01 05:30:00.000000000 +0530
+++ y/etc/apt/sources.list.d/ubuntu.sources	2025-01-03 06:53:06.000000000 +0530
@@ -0,0 +1,45 @@
+# See http://help.ubuntu.com/community/UpgradeNotes for how to upgrade to
+# newer versions of the distribution.
+
+## Ubuntu distribution repository
+##
+## The following settings can be adjusted to configure which packages to use from Ubuntu.
+## Mirror your choices (except for URIs and Suites) in the security section below to
+## ensure timely security updates.
+##
+## Types: Append deb-src to enable the fetching of source package.
+## URIs: A URL to the repository (you may add multiple URLs)
+## Suites: The following additional suites can be configured
+##   <name>-updates   - Major bug fix updates produced after the final release of the
+##                      distribution.
+##   <name>-backports - software from this repository may not have been tested as
+##                      extensively as that contained in the main release, although it includes
+##                      newer versions of some applications which may provide useful features.
+##                      Also, please note that software in backports WILL NOT receive any review
+##                      or updates from the Ubuntu security team.
+## Components: Aside from main, the following components can be added to the list
+##   restricted  - Software that may not be under a free license, or protected by patents.
+##   universe    - Community maintained packages.
+##                 Software from this repository is only maintained and supported by Canonical
+##                 for machines with Ubuntu Pro subscriptions. Without Ubuntu Pro, the Ubuntu
+##                 community provides best-effort security maintenance.
+##   multiverse  - Community maintained of restricted. Software from this repository is
+##                 ENTIRELY UNSUPPORTED by the Ubuntu team, and may not be under a free
+##                 licence. Please satisfy yourself as to your rights to use the software.
+##                 Also, please note that software in multiverse WILL NOT receive any
+##                 review or updates from the Ubuntu security team.
+##
+## See the sources.list(5) manual page for further settings.
+Types: deb
+URIs: http://ports.ubuntu.com
+Suites: oracular oracular-updates oracular-backports
+Components: main universe restricted multiverse
+Signed-By: /usr/share/keyrings/ubuntu-archive-keyring.gpg
+
+## Ubuntu security updates. Aside from URIs and Suites,
+## this should mirror your choices in the previous section.
+Types: deb
+URIs: http://ports.ubuntu.com
+Suites: oracular-security
+Components: main universe restricted multiverse
+Signed-By: /usr/share/keyrings/ubuntu-archive-keyring.gpg

--- x/etc/adduser.conf	2024-08-26 14:45:01.000000000 +0530
+++ y/etc/adduser.conf	2025-01-03 06:49:13.000000000 +0530
@@ -97,13 +97,13 @@
 # list of groups that new non-system users will be added to
 # if ADD_EXTRA_GROUPS is non-zero or set on the command line.
 # Default: EXTRA_GROUPS="users"
-#EXTRA_GROUPS="users"
+EXTRA_GROUPS="video"

 # Setting this to something other than 0 will cause adduser to add
 # newly created non-system users to the list of groups defined by
 # EXTRA_GROUPS.
 # Default: ADD_EXTRA_GROUPS=0
-#ADD_EXTRA_GROUPS=0
+ADD_EXTRA_GROUPS=1

 # use extrausers by default
 #USE_EXTRAUSERS=1

--- x/etc/kernel/cmdline	1970-01-01 05:30:00.000000000 +0530
+++ y/etc/kernel/cmdline	2025-01-03 06:49:13.000000000 +0530
@@ -0,0 +1 @@
+rootwait rw console=ttyS2,1500000 console=tty1 cgroup_enable=cpuset cgroup_memory=1 cgroup_enable=memory
\ No newline at end of file

--- x/usr/share/u-boot-menu/conf.d/ubuntu.conf	1970-01-01 05:30:00.000000000 +0530
+++ y/usr/share/u-boot-menu/conf.d/ubuntu.conf	2025-01-03 06:49:13.000000000 +0530
@@ -0,0 +1,3 @@
+U_BOOT_PROMPT="1"
+U_BOOT_PARAMETERS="$(cat /etc/kernel/cmdline)"
+U_BOOT_TIMEOUT="20"

